test/common/testlib.py

```diff
@@ -755,6 +755,7 @@ def assert_pixels(self, selector, key, ignore=[]):
         if not (Image and self.pixels_label and self.cdp and self.cdp.valid):
             return
 
+        self.call_js_func('ph_wait_for_animations_to_finish', selector)
         self.call_js_func('ph_scrollIntoViewIfNeeded', selector)
```

Comment:
Should we swap these lines? Intuitively it could happen that scrolling to make an element visible will trigger further animations?

Reply:
Ok, done.